base_layer/wallet_ffi/src/lib.rs

```diff
@@ -2916,22 +2935,23 @@ pub unsafe extern "C" fn liveness_data_get_public_key(
 /// # Safety
 /// The ```liveness_data_destroy``` method must be called when finished with a TariContactsLivenessData to prevent a
 /// memory leak
+
 #[no_mangle]
 pub unsafe extern "C" fn liveness_data_get_latency(
     liveness_data: *mut TariContactsLivenessData,
     error_out: *mut c_int,
-) -> c_int {
+) -> c_uint {
     let mut error = 0;
     ptr::swap(error_out, &mut error as *mut c_int);
     if liveness_data.is_null() {
         error = LibWalletError::from(InterfaceError::NullError("liveness_data".to_string())).code;
         ptr::swap(error_out, &mut error as *mut c_int);
-        return -1;
+        return 0;
```

Comment:
The docs for the function state:
`value of '-1' if it is None. Note that it also returns '-1' if liveness_data is null.`

This may effect mobile code, and makes the docs out of date.

Reply:
updated the docs, felt that a latency of 0ms is more unlikely as a latency wrap-around. It also stop the check of checking for <0 rather than == -1, which should now by == 0